test_data.rb

```diff
@@ -199,51 +191,13 @@ def test_unique_suggestable_name
     names = Set.new
```

Comment:
Shouldn't you have two sets now? One to test `name` and one to test `name + country`?

Reply:
Both work, but I had amend the commit